app/jobs/slack_event/admins_job.rb

```diff
@@ -0,0 +1,70 @@
+#
+# This comes through the slash commands API which has a
+# slightly different payload than the events API
+#
+# Example payload:
+#
+# {
+#   "token": "xxxx",
+#   "team_id": "TCF0ZD3LL",
+#   "team_domain": "railslink-dev",
+#   "channel_id": "xxxx",
+#   "channel_name": "xxxx",
+#   "user_id": "xxxx",
+#   "user_name": "xxxx",
+#   "command": "/admins",
+#   "text": "",
+#   "response_url": "https://hooks.slack.com/commands/TCF0ZD3LL/640895425681/GDK2vHvOpxpvFrpRJnrvxiHS",
+#   "trigger_id": "xxxx",
+#   "slack": {
+#     "event": {
+#       "type": "admins"
+#     }
+#   },
+#   "controller": "slack",
+#   "action": "event"
+# }
+#
+
+class SlackEvent::AdminsJob < ApplicationJob
+
+  attr_reader :options
+  private :options
+
+  def perform(options = {})
+    @options = options.with_indifferent_access
+    message = {
+      text: admin_text,
+      response_type: "in_channel"
```

Comment:
I noticed that I can see your existing tests of this command in the #random channel of the rubyonrailslink-dev workspace.  I suspect we don't want it publicly shown, but only shown to the requestor.  I think that's called "ephemeral" (it is for other slack stuff anyway).  Can we make that tweak?

Reply:
This is what those types of messages look like:
![image](https://user-images.githubusercontent.com/6721355/58647553-74bd9980-82cd-11e9-84e7-9599c08213e8.png)

Reply:
@katelovescode can you change this to "ephemeral" and push? I'll approve and squash and merge.  I've tested it works.